src/OperationKeyboard/controller.js

```diff
@@ -0,0 +1,47 @@
+"use strict";
+
+import { EventDispatcher } from '../eventDispatcher.js';
+import viewTemplate from './view.hbs';
+import stringToDom from 'string-to-dom';
+
+export const OperationKeyboardEvents = {
+    OPERATION_SELECTED: 'operationSelected'
+};
+
+export const Operations = {
+    ADDITION: '+',
+    SUBSTRACTION: '-',
+    DIVISION: '/',
+    MULTIPLICATION: 'x',
+    EQUALLS: '='
+};
+
+export class OperationKeyboard extends EventDispatcher {
+
+    /**
+     *  Constructor.  
+     */
+    constructor() {
+        super();
+        this._bindOpsEvents();
+    }
+    /**
+     * Binds event listeners to operation buttons.
+     */
+    _bindOpsEvents() {
+        let viewString = viewTemplate();
+        this.view = stringToDom(viewString);
+        let operationButtons = this.view.querySelectorAll('.operation');
+        let operation = null;
+        for (let i = 0; i < operationButtons.length; i++) {
+            operationButtons[i].addEventListener("click", (event) => {
+                for(let j in Operations) {
```

Comment:
Using of `i` and `j` make sense when you are dealing with indexes. But here, you are iterating over object fields, so it would be more readable to do:
```
for (let operation in Operations) {
```